examples/weather/graphcast/train_graphcast.py

```diff
@@ -25,11 +25,18 @@
 import torch.cuda.profiler as profiler
 from torch.optim.lr_scheduler import SequentialLR, LinearLR, CosineAnnealingLR, LambdaLR
 
+import torch._dynamo
+
+torch._dynamo.config.suppress_errors = True
```

Comment:
What do we need this for?

Reply:
Without this, torch dynamo gives an error on the machine I'm using for debugging. I'll put a note to check later if this can be removed.